packages/spectacle/package.json

```diff
@@ -129,37 +112,13 @@
       ]
     },
     "build:dist": {
-      "dependencies": [
-        "build:dist:dev",
```

Comment:
We don't want to publish a non-minified UMD anymore? 

Side note -- we presently _don't_ publish sourcemaps anywhere -- so usually if we're doing only min we may want to consider publishing those...

Reply:
Oh good point. Let me add that back. Do we have any use case for UMD builds outside of one page?

Reply:
Not presently that I know -- and once import maps truly land in browsers then we can finally switch over to just our ESM version.

Reply:
I was struggling to make it work using the `iffe` version `tsup` can build, but we really need some of the polyfills our webpack provides (like `process/browser`) to make this work in the browser.